.github/workflows/e2e-tools.yml

```diff
@@ -47,12 +47,12 @@ jobs:
       run: |
         # kind is part of 20.04 image, see: https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md
         kind version
-        kind create cluster --config=hack/kind-config-e2e-no-registry.yaml --image kindest/node:v1.21.1@sha256:69860bda5563ac81e3c0057d654b5253219618a22ec3a346306239bba8cfa1a6
+        kind create cluster --config=hack/kind-config-e2e-no-registry.yaml --image kindest/node:v1.24.0@sha256:0866296e693efe1fed79d5e6c7af8df71fc73ae45e3679af05342239cdc5bc8e
```

Comment:
will you squash the commits or do you want this visible? if visible may you tell me why please?

Reply:
I'm not squashing because this commit is logically different and deserve its own autonomy.
To do things _really_ by the book, I should have sent a separate PR with the fixes to the existing configuration, wait for it to be merged, only then rebase this one on top of that and get it LGTM'd.